psi4/src/psi4/libscf_solver/rohf.cc

```diff
@@ -844,22 +774,22 @@ void ROHF::Hx(SharedMatrix x, SharedMatrix ret)
     ret->add(Hx_right);
     ret->scale(4.0);
 
-    Hx_left.reset(); Hx_right.reset();
-    Cocc.reset(); Cvir.reset();
+    Hx_left.reset();
+    Hx_right.reset();
+    Cocc.reset();
+    Cvir.reset();
 }
 
-void ROHF::damp_update()
-{
-  Da_->scale(1.0 - damping_percentage_);
-  Da_->axpy(damping_percentage_, Da_old_);
-  Db_->scale(1.0 - damping_percentage_);
-  Db_->axpy(damping_percentage_, Db_old_);
-  Dt_->copy(Da_);
-  Dt_->add(Db_);
+void ROHF::damping_update(double damping_percentage) {
+    Da_->scale(1.0 - damping_percentage);
+    Da_->axpy(damping_percentage, Da_old_);
+    Db_->scale(1.0 - damping_percentage);
+    Db_->axpy(damping_percentage, Db_old_);
+    Dt_->copy(Da_);
+    Dt_->add(Db_);
 }
 
-int ROHF::soscf_update()
-{
+int ROHF::soscf_update(float soscf_conv, int soscf_min_iter, int soscf_max_iter, int soscf_print) {
```

Comment:
Ditto from above.

Reply:
There aren't two memset pairs in rohf, but I went ahead and applied pattern to cuhf, rhf, rohf, uhf